tests/test_generator.py

```diff
@@ -265,6 +267,43 @@ def test_multiple_tags(fileutils):
         )
 
 
+def test_generation_log(fileutils):
+    """Test that an error is issued when a tag is unused and make_fatal is True"""
+
+    test_dir = fileutils.make_test_dir()
+    exp = Experiment("gen-log-test", test_dir, launcher="local")
+
+    params = {"THERMO": [10, 20], "STEPS": [10, 20]}
+    ensemble = exp.create_ensemble("dir_test", params=params, run_settings=rs)
+    conf_file = get_gen_file(fileutils, "in.atm")
+    ensemble.attach_generator_files(to_configure=conf_file)
+
+    def not_header(line):
+        """you can add other general checks in here"""
+        if line.startswith("Generation start date and time:"):
+            return False  # ignore it
+        return True
```

Comment:
If you want to save a couple of lines
```py
return not line.startswith(...)
```

Reply:
Let's save lines, in case we ever print everything to paper.